### PR TITLE
Adding `cc!notes @user` command

### LINKED ISSUE #41: Replacing moderation commands warn and infractions
I'd like the commands warn and infractions to be replaced with our own custom commands

#### Command Format
```
cc!warn @ user reason
cc!infractions @ user
cc!addnote @user usernote
cc!notes @user
```
#### Command Usage
The warn commands sends the user a DM saying the reason they've been warned and that they have been. No further action is taken except what is specified in #7 
Pulls all the infractions of a certain user from the database and prints them to the channel in which the command was made.
The addnote command adds a note (a string of text) to a specific user. It should not contact the user who the note was added to in anyway. This should not count as an infraction and should only be logged to the mod_logs table.
The notes command will just display all the notes that have been added to a user.

#### Limited Usage
All of these commands are limited to Super Users, Mods, and Admins

Check issue #7 for more information.

### PR DESCRIPTION
**Pull request by fedGL**
## What issue is this solving?
Closes #41 and continues push for issue #7 

### Description
Fourth and final command from the infractions and notes family of commands. This is a refactoring of the very functionally-similar `cc!infractions @user` command. It should work out the box.

## Any helpful knowledge/context for the reviewer?

- Is a re-seeding of the database necessary? No
- Any new dependencies to install? No
- Any special requirements to test? Admin/Mod/SU
  - (e.g., admin perms, alt account, etc.)

### Please make sure you've attempted to meet the following coding standards
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code

### CODE REVIEW

**Review by aedwardg — CHANGES_REQUESTED** [review]
Great job @fedGL!  

Just two tiny changes and two comments about potential future restructuring (for separate issues/PRs).

File: commands/moderation/notes.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+module.exports = {
+  name: 'notes',
+  description: 'finds user notes record in db and returns it to channel',
+  execute(msg, con, args) {
+    // Make sure only SU, Mods and Admin can run the command
+    const targetUser =
+      msg.mentions.members.first() || msg.guild.members.cache.get(args[0]);
+
+    if (targetUser) console.log(`targerUser is true`);
```
[the remark this answers is not in the document]
Comment (fedGL) [reply commands/moderation/notes.js:11]: Done, `console.log` removed

File: commands/moderation/notes.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  con.query(`${sqlNotes}`, function (err, result) {
+    if (err) {
+      console.log(err);
+      // Include a warning in case something goes wrong writing to the db
+      msg.channel.send(
+        `I couldn't read ${targetUser}'s notes from the db!`
+      );
+    } else {
+      console.log('Found note records.');
+      notesLog(msg, targetUser, result);
```
[the remark this answers is not in the document]
Comment (fedGL) [reply commands/moderation/notes.js:34]: Check 👍

File: commands/moderation/notes.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    } else if (Math.floor(hours) > 0) {
+      timeSinceNote[idx] = [
+        `${Math.floor(hours)}h ${Math.round(hours % 60)}m`,
+      ];
+    } else {
+      timeSinceNote[idx] = [
+        `${Math.floor(minutes)}m ${Math.round(minutes % 60)}s`,
+      ];
+    }
+  });
```
[the remark this answers is not in the document]
Comment (fedGL) [reply commands/moderation/notes.js:112]: Noted 👍 We can create a new issue to refactor reused coded into another place

File: commands/moderation/notes.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    return true;
+  }
+}
+
+function hasUserTarget(msg, targetUser) {
+  // Asortment of answers to make the bot more fun
+  const failAttemptReply = [
+    'Ok there bud, whose notes are you trying to check again?',
+    'You definitely missed the targer user there...',
+    'what? You want ALL the notes from everyone? You forgot the targer user',
```
[the remark this answers is not in the document]
Comment (fedGL) [reply commands/moderation/notes.js]: Fixed

**Review by aedwardg — APPROVED** [review]
LGTM! 🚀

**Review by Vic-ST — APPROVED** [review]
LGTM!

**Pull request merged**